pkg/logs/input/docker/launcher.go

```diff
@@ -169,3 +175,45 @@ func (l *Launcher) stopTailer(containerID string) {
 		delete(l.tailers, containerID)
 	}
 }
+
+func (l *Launcher) restartTailer(containerID string) {
+	backoffDuration := 1 * time.Microsecond
```

Comment:
I don't think that a `Microsecond` is a good reference here, I think we should use a `Second` instead, WDYT ?

Reply:
Yes, I was actually using `Second` but I switch to `Microsecond` to manually test the retry logic (because the docker socket restarts really quickly)